packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -44,6 +44,9 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @notice Version identifier, used for upgrades.
     uint256 public constant VERSION = 0;
 
+    /// @notice The number of decimals that the gas paying token has.
+    uint8 public constant GAS_PAYING_TOKEN_DECIMALS = 18;
```

Comment:
We should be careful about exposing this in the public ABI. This is fine to use as an internal constant but we don't want to make it public